openfecwebapp/api_caller.py

```diff
@@ -86,14 +86,19 @@ def load_legal_advisory_opinion(ao_no):
     if not (documents and len(documents)):
         return None
 
-    canonical_document = documents[0]
+    for document in documents:
+        canonical_document = document
+        if (document['category'] == 'Final Opinion'):
+            break        
+    
```

Comment:
Also to the other question @ccostino, yes if there is no `Final Opinion` document, we will are purposedly ending up on the last document in the array. (Hence the changes to use `category` as the button label in `legal-advisory-opinion.html`